src/hotspot/share/nmt/memMapPrinter.cpp

```diff
@@ -203,6 +204,8 @@ static void print_thread_details(uintx thread_id, const char* name, outputStream
 // Given a region [from, to), if it intersects a known thread stack, print detail infos about that thread.
 static void print_thread_details_for_supposed_stack_address(const void* from, const void* to, outputStream* st) {
 
+  ResourceMark rm;
```

Comment:
Why was this needed to be added here? Shouldn't callers have any needed ResourceMarks?

Reply:
Okay. I was trying to find where the existing callers have their RM and it is not obvious.

Reply:
Existing caller is DCmd::parse_and_execute, where we have an RM that guards every individual Dcmd execution.